NEWS.md

```diff
@@ -24,6 +24,10 @@ v1.4.0, TBD
 - Add support for multi-iov RMA read and aromic operations
 - Allow multi-recv to post buffer larger than message size limit
 
+## RxD provider notes
+
+- Initial release
```

Comment:
All utility providers have been disabled for now, until after the 1.4 release.  If you think rxd is ready, it willl need to be re-enabled.